Userland/Applications/FileManager/FileOperationProgressWidget.cpp

```diff
@@ -92,7 +115,7 @@ void FileOperationProgressWidget::did_finish()
     window()->close();
 }
 
-void FileOperationProgressWidget::did_error(const String message)
+void FileOperationProgressWidget::did_error(String const message)
```

Comment:
This `String const` without a reference is really weird. If you're touching it anyways, you could change the argument type to `StringView const&` and change the callers from `substring` to `substring_view`. But of course you only changed the `const`, so no obligation to do so. :^)

Reply:
Sure, I'll keep an eye out for that in the future. (const and references are both fairly new to me.)